subworkflows/local/discovery/segemehl.nf

```diff
@@ -0,0 +1,27 @@
+include { SEGEMEHL_INDEX as INDEX   } from '../../../modules/nf-core/segemehl/index'
+include { SEGEMEHL_ALIGN as ALIGN   } from '../../../modules/nf-core/segemehl/align'
+include { GAWK as UNIFY             } from '../../../modules/nf-core/gawk'
+
+
+workflow SEGEMEHL {
+    take:
+    reads
+    fasta
+    index
+
+    main:
+    ch_versions = Channel.empty()
+
+    index = index ?: INDEX( fasta ).index
```

Comment:
Look here

Reply:
Since the segemehl index is only used in the segemehl subworkflow I thought it might be more understandable putting it right here